lib/iris/tests/unit/io/test_expand_filespecs.py

```diff
@@ -96,6 +96,12 @@ def test_files_and_none(self):
 
         self.assertMultiLineEqual(str(err.exception), expected)
 
+    def test_false_bool(self):
```

Comment:
Could we split this into something equivalent to `test_absolute_path()` and `test_relative_path()`? Just to confirm that the expansion functionality still works with `files_expected=False`.